chapters/en/slides/module2/module2_02.Rmd

```diff
@@ -0,0 +1,279 @@
+
+---
+
+## Using `info` to view dataframe columns
+
+```{python}
+gm.info()
+```
+
+Notes:
+When reading in a new dataset,
+it is always a good idea to have a look at a few rows
+like we did in the previous slide
+to get an idea of how the data looks.
+
+Another helpful practice is to use the `.info()` method
+to get an overview of the column types
+and see if there are any `NaNs` (missing values).
+
+---
+
+## A line chart is a good choice for trends over time
+
+```{python}
+import altair as alt
+
+
+alt.Chart(gm).mark_line().encode(
+    x='year',
+    y='population')
+```
+
+Notes:
+Since the data reaches all the way back to the 1800s,
+it would be really interesting to plot
+how the world population have been growing up until today.
+We could use a line plot for this
+as we learned in the first module.
+
+Why does the plot look so weird?
+Because for every year in the dataset,
+we have plotted the population for every single country
+and connected them with a vertical line.
+That's not what we want!
+
+---
+
+## The `sum` aggregate method will show us the trend of the entire world
+
+```{python}
+alt.Chart(gm).mark_line().encode(
+    x='year',
+    y='sum(population)')
+```
+
+Notes:
+We want to see how the **world** population has been changing.
+We can calculate this by summing up all the countries'
+populations for each year via the `sum` aggregation method in Altair.
+
+That looks much better!
+Interestingly,
+it seem like the world population have been growing in two distinct phases.
+We will talk more about that in a few slides,
+but first,
+let's recreate this chart as an area chart.
+
+---
+
+## Area charts are also effective at visualizing trends over time
+
+```{python}
+alt.Chart(gm).mark_area().encode(
+    x='year',
+    y='sum(population)')
+```
+
+Notes:
+Thanks to Altair's consistent visualization grammar,
+the modification we need to make to the code are minor.
+Replace `mark_line` with `mark_area` and we're done.
+
+For showing a singe trend over time,
+the choice between a line and area chart comes down to asthetics.
+They are both effective choices for communicating
+
+However,
+when visualizing the trends over time for multiple groups,
+lines and areas have different advantages,
+as we will see next.
+
+---
+
+## Area charts are preferred when the total of the groups is the most important
+
+```{python}
+alt.Chart(gm).mark_area().encode(
+    x='year',
+    y='sum(population)',
+    color='region')
+```
+
+Notes:
+The y-axis range of this chart
+is the same as the one on the previous slide,
+so it is easy to see how the total world population has changed over time.
+
+From the stacked colored regions,
+we also get a rough idea of how each region has grown,
+but it is hard to compare exactly,
+especially for regions that are not stacked next to each other.
+For example,
+we can't really tell if Europe, Africa, or the Americas
+has the largest population in the most recent year.
+
```

Comment:
Add a summary sentence here with that take home for what this plot is good for, and when to use it (ending on what its problems are makes it seem likes it not a great plot type).